kanidm_client/src/asynchronous.rs

```diff
@@ -835,7 +835,7 @@ impl KanidmAsyncClient {
         attr: &str,
         values: &[&str],
     ) -> Result<bool, ClientError> {
-        let m: Vec<_> = values.iter().map(|v| (*v).to_string()).collect();
+        let msg: Vec<_> = values.iter().map(|v| (*v).to_string()).collect();
         self.perform_post_request(format!("/v1/account/{}/_attr/{}", id, attr).as_str(), m)
```

Comment:
```suggestion
        let msg: Vec<String> = values.iter().map(str::to_string).collect();
        self.perform_post_request(format!("/v1/account/{}/_attr/{}", id, attr).as_str(), msg)
```

Reply:
It's not letting me apply this, but it's an easy manual change I'll just push it.

Reply:
Ahh it doesn'th ork because it's an &&str here because of iter over &str. We'd need into iter maybe. Anyway, ignoring for now, as this is a minor nit.